pybossa/jobs.py

```diff
@@ -550,10 +550,9 @@ def disable_users_job():
     ext_user_domains = current_app.config.get('EXTENDED_STALE_USERS_DOMAINS') or []
```

Comment:
I cannot leave a comment on it but the line above this at 549, should I remove it?

Reply:
It may be useful to keep this value as-is (i.e., not remove it). Since there are other config setting values, and they're each optional, this makes it easy to configure the settings in the future. If we need to circle back and change this, we would already have a config setting ready to go.

It could also be helpful to add a comment above each of those 3 lines to indicate what those settings are configuring. Thank you.

Reply:
In that case this is all ready @ywrsusan